highfidelity/hifi
# Initial 180 flip

Places the Semi-billboarded preview overlay on top of the current spectator camera instead of behind, and causes the overlay to flip 180 degrees around the y-axis when the user's avatar has passed the spectator camera.

To Test:
-Make sure that the overlay is on top of the spectator camera (red box placeholder)
-Zoom out of your avatar to Third Person, so you can see both your avatar and the spectator camera
-Walk past the spectator camera, and check to see that the overlay flips
